lib/src/building_explorer/building_explorer.dart

```diff
@@ -0,0 +1,60 @@
+//
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+part of '../../arcgis_maps_toolkit.dart';
+
+class BuildingExplorer extends StatefulWidget {
```

Comment:
This can be a `StatelessWidget`, I think. Generally we should probably prefer `StatelessWidget` unless there's a reason `StatefulWidget` is necessary.